content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,19 @@
+title: "A Statement on Richard Stallman"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+_A Statement from Mixxx on Richard Stallman's return to the FSF._
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a partial acknowledgement of the [harm he's caused over multiple decades](https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88), including creating a sexually hostile work and learning environment, sexual harassment, and abuse of power.
+Loss of his position in the community was only the barest of consequences, but his resignation came as a relief for many in the community who have suffered his bad behavior.
+
+While Mixxx is not officially associated with the Free Software Foundation,
+there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
+Therefore we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation not only of Richard Stallman, but the entire FSF board of directors.
+By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.
```

Comment:
Furthermore, we call upon the Free Software Foundation to revise its bylaws to democratic governance processes for choosing leadership.
We believe that the myriad of software developers who chose to trust the FSF with stewardship of licenses for the code we wrote deserve to have input over the leadership of the FSF.

Reply:
added (slightly tweaked)